drafting/P2968Rx_ignore.md

```diff
@@ -43,31 +43,11 @@ To summarize the proposed change:
 1. better self-documenting code telling the intent
 2. Improved teachability of C++ to would-be C++ programmers in safety critical environments
 
-## Comparison Table
```

Comment:
I would love to keep this table

Reply:
This is one of the classic examples of "not a good Tony Table" in... well, somewhere I can't find again right now. But whoever-it-was was very emphatic that if your entire Tony Table is "BEFORE: does not compile. AFTER: does compile" — that's Not Doing It Right.

The "BEFORE" should show what people were actually required to write, before the proposal — the thing that they can now _replace_ with what's on the right-hand side. (In this specific case, it would be `(void)std::printf("hello ignore\n");` or simply `printf("hello ignore\n");`, which is not actually good marketing for this proposal. I have no concrete suggestions for how to salvage the table.)